Ginger/Ginger/ALM/Qtest/QtestCyclesExplorerPage.xaml.cs

```diff
@@ -0,0 +1,239 @@
+            if (testSetItem.AlreadyImported)
+            {
+                lblTSAlreadyImported.Content = "Yes";
+                txtBoxTSMappedBF.Text = "'" + testSetItem.MappedBusinessFlowPath + "'";
+            }
+            else
+            {
+                lblTSAlreadyImported.Content = "No";
+                txtBoxTSMappedBF.Text = "None";
+            }
+        }
+
+        private void ShowTestSetDetailsPanel(bool toShow)
+        {
+            if (toShow)
+            {
+                TestSetDetailsPanel.Visibility = System.Windows.Visibility.Visible;
+                testSetDetailsColumn.Width = new GridLength(200);
+            }
+            else
+            {
+                TestSetDetailsPanel.Visibility = System.Windows.Visibility.Collapsed;
+                testSetDetailsColumn.Width = new GridLength(0);
+            }
+        }
+
+        public object ShowAsWindow(eWindowShowStyle windowStyle = eWindowShowStyle.Dialog)
+        {
+            if (mParentSelectionMode)
+            {
+                Button importBtn = new Button();
+                importBtn.Content = "Select";
+                importBtn.Click += new RoutedEventHandler(SelectFolder);
+                GingerCore.General.LoadGenericWindow(ref _GenericWin, App.MainWindow, windowStyle, "Select Path For Export", this, new ObservableList<Button> { importBtn }, true, "Cancel", Cancel_Clicked);
+                return mCurrentSelectedObject;
+            }
+            else
+            {
+                Button importBtn = new Button();
+                importBtn.Content = "Import Selected";
+                importBtn.Click += new RoutedEventHandler(ImportSelected);
+                GingerCore.General.LoadGenericWindow(ref _GenericWin, App.MainWindow, windowStyle, "Browse Qtest Cycles", this, new ObservableList<Button> { importBtn }, true, "Cancel", Cancel_Clicked);
+                return CurrentSelectedTestSuites;
+            }
+        }
+
+        private void SelectFolder(object sender, RoutedEventArgs e)
+        {
+            if (mCurrentSelectedTreeItem == null)
+                Reporter.ToUser(eUserMsgKey.StaticWarnMessage, "Please select a folder item");
+            else
+            {
+                _GenericWin.Close();
+            }
+        }
+
+        private void ImportSelected(object sender, RoutedEventArgs e)
+        {
+            if (mCurrentSelectedTreeItem != null)
+            {
+                if (ALMIntegration.Instance.ImportSelectedTestSets(mImportDestinationPath, (IEnumerable<object>)CurrentSelectedTestSuites) == true)
+                {
+                    //Refresh the explorer selected tree items import status
+                    LoadDataBizFlows();
+                    //foreach (QCTestSetTreeItem testSet in CurrentSelectedTestSets)
+                    //    testSet.IsTestSetAlreadyImported();
+                    ShowTestSetDetailsPanel(false);
+                }
+            }
+            else
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'else' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)